tooling/scripts/test-categories.yaml

```diff
@@ -32,6 +32,7 @@ group-02:
   - elasticsearch-rest
   - google-bigquery
   - google-pubsub
+  - google-storage
```

Comment:
group-01 or group-08 may achieve a greater balance https://github.com/apache/camel-quarkus/actions/runs/773244432

Reply:
Sounds correct, thanks for the link

Reply:
@aldettinger  I moved test into group-01